src/strings/csv/es.csv

```diff
@@ -566,14 +575,18 @@ NOTES,Notas
 NOTIFICATIONS,Notificaciones
 NUMBER_OF_PACKETS,Número de paquetes
 NUMBER_OF_PEOPLE,Número de personas
+NUMBER_OF_PLANTS,Número de plantas
 NUMBER_OF_PLANTS_BY_SPECIES,Número de plantas por especie
 NUMBER_OF_PLANTS_BY_SUBZONE_AND_SPECIES,Número de plantas por subzona y especie
-NUMBER_OF_SEEDS_COMPROMISED,# Semillas Afectadas
+NUMBER_OF_SEEDS_COMPROMISED,Número de semillas afectadas
```

Comment:
this is a breaking change, I will update the components to use _V2 which was reintroduced with the previously correct value